tests/run/worktree/test.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "tmp=\$(mktemp -d)" 0 "Creating tmp directory"
+        rlRun "cp -rf data/* $tmp"
```

Comment:
Do we need to copy the files? The test does not store any files in the `data` directory.

Reply:
yeah, no files created, so no